packages/cxl-ui/scss/global/cxl-marketing-nav.scss

```diff
@@ -86,3 +91,29 @@ vaadin-context-menu-list-box {
     max-width: var(--cxl-vaadin-context-menu-item-max-width);
   }
 }
+
+/**
+ * Mobile view `vaadin-context-menu-item` sub menu back button.
+ *
+ * @since 2022.11.10
+ * @see https://app.clickup.com/t/3rgnekt
+ */
+/* stylelint-disable-next-line selector-no-qualifying-type */
+.context-menu-item-back-button {
+  min-width: auto;
+  height: calc(var(--lumo-button-size) / 2);
+  padding: 0;
+  margin: 0;
+  line-height: 1;
+  background-color: transparent;
+}
+
+.context-menu-item-back-button iron-icon {
+  margin-left: calc(var(--lumo-icon-size-m) / 4 * -1);
+}
+
+@media (min-width: 420px) and (min-height: 420px) {
+  .back-button-menu-item {
+    display: none;
+  }
+}
```

Comment:
Was it not possible to avoid global scope here?

Reply:
> Was it not possible to avoid global scope here?

I didn't know how I can dynamically inject the css at runtime into `vaadin-context-menu-item`, as we are not overriding it in our code, so global styles made sense to me.

Any suggestion?